code/__HELPERS/_lists.dm

```diff
@@ -485,6 +485,7 @@
 
 //Picks from the list, with some safeties, and returns the "default" arg if it fails
 #define DEFAULTPICK(L, default) ((islist(L) && length(L)) ? pick(L) : default)
+#define THISORTHAT(THIS, THAT) (THIS ? THIS : THAT)

```

Comment:
why on earth do we need a define for an operator

Reply:
and what is it doing in the list defines

Reply:
ALSO just fucking use `this || that` lmao
but not as a define that's dumb

Reply:
to understand this you have to understand the thought proces.s.

I wanted the equivalent of defaultpick, but for just accessing lists, but then i went to find the usage to add it in, and i noticed that it didn't actually access an index it passed it to istypeintypecache, so i made it for reading a variable, not accessing an index.

Reply:
it's clearer

Reply:
anywho, I plan on keeping it, it's easier to read and follow chains of this compared to ? that I actually want to ban the use of ? now and make everything use thisorthat

Reply:
use `this || that` since it accomplishes what you want and kill this